src/commands/package-and-push.yaml

```diff
@@ -5,25 +5,43 @@ parameters:
     type: "string"
   chart:
     type: "string"
+  on_tag:
```

Comment:
Now I have slept with that thought I think `on_branch` which is `""` by default would make more sense. WDYT? Do you still have any energy left to change that?

Reply:
Sorry, I almost missed this comment. I'm okay with changing it although the value seems marginal with our current use cases. I'll put it on my list as a lower priority item.

Reply:
Feel free to ignore.